src/sparseml/transformers/masked_language_modeling.py

```diff
@@ -50,14 +52,20 @@
 )
 from transformers.trainer_utils import get_last_checkpoint
 from transformers.utils import check_min_version
+from transformers.utils.versions import require_version
 
 from sparseml.transformers.sparsification import Trainer
 from sparseml.transformers.utils import SparseAutoModel
 
 
 # Will error if the minimal version of Transformers is not installed.
 # Remove at your own risks
-check_min_version("4.7.0.dev0")
+check_min_version("4.18.0.dev0")
+
+require_version(
+    "datasets>=1.8.0",
```

Comment:
This should be 1.18.0.

Reply:
@spacemanidol This is taken from the official transformers repo, which currently enforces `>=1.8.0`. Do our use cases require `>=1.18.0`?

Reply:
yes. the conll datasets and a bunch others have broken links with previous versions